Modules/Loadable/Segmentations/EditorEffects/qSlicerSegmentEditorScriptedEffect.h

```diff
@@ -119,6 +119,8 @@ class Q_SLICER_SEGMENTATIONS_EFFECTS_EXPORT qSlicerSegmentEditorScriptedEffect
   void layoutChanged() override;
   /// Let the effect know that the interaction node is modified
   void interactionNodeModified(vtkMRMLInteractionNode* interactionNode) override;
+  /// Release any resources, event observers, Qt signal / slot connections before calling delete
```

Comment:
```suggestion
  /// Clean up resources, event observers, and Qt signal/slot connections before deletion.
```

I also though about using 

```
  /// \copydoc qSlicerSegmentEditorScriptedEffect::cleanup()
```

.. but this is not done anywhere else.
See https://www.doxygen.nl/manual/commands.html#cmdcopydoc